packages/reactive-element/src/reactive-element.ts

```diff
@@ -737,7 +737,7 @@ export abstract class ReactiveElement extends HTMLElement {
    * overridden on the element instance with a function that triggers the first
    * update.
    */
-  protected enableUpdating() {}
+  protected enableUpdating(_value: boolean) {}
```

Comment:
why was this needed? Can you make it optional to leave out the `true` constant above?

Reply:
Done.